builtin/settingtypes.txt

```diff
@@ -508,6 +508,9 @@ directional_colored_fog (Colored fog) bool true
 #    set to the nearest valid value.
 ambient_occlusion_gamma (Ambient occlusion gamma) float 2.2 0.25 4.0
 
+#    Enables inventory items animations.
+enable_inventory_animations (Inventory items animations) bool false
```

Comment:
Other bools in this file do not use the 'enable' word, so (Animate inventory items).